scripts/update_version_table.py

```
@@ -0,0 +1,131 @@
+        return "N/A"
+
+
+def extract_kitsune2_version(tag):
+    content = git_show(tag, "Cargo.lock")
+    if content is None:
+        return "N/A"
+    try:
+        for package in tomllib.loads(content).get("package", []):
+            if package.get("name") == "kitsune2":
+                return package.get("version", "N/A")
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)